test/debug.ts

```diff
@@ -333,24 +333,4 @@ describe("debug command tests", () => {
 			});
 		});
 	});
-
-	describe("Debugger tests", () => {
-		let testInjector: IInjector;
-
-		beforeEach(() => {
-			testInjector = createTestInjector();
-		});
-
-		it("Ensures that beforePrepareAllPlugins will call gradle with clean option when *NOT* livesyncing", async () => {
```

Comment:
I believe we should keep this test - we still have a case where we want to call gradle clean (using the old runtime logic).

Reply:
Yes. you are right, do you think there is a straightforward way to disable it/always assert true when a more recent runtime is used? Or should I use the same logic as in - https://github.com/NativeScript/nativescript-cli/pull/3032/files/f4a47a233f86e8d499825247342ce98b1404d525#diff-b64860922c31aeed973aef6663b2a536R95? Thank you!

Reply:
I would modify the test and make it always use runtime version 3.2.0. This way you'll not have to handle the new scenario, just ensure we call clean.